ma_gym/envs/combat/combat.py

```diff
@@ -45,38 +45,47 @@ class Combat(gym.Env):
     metadata = {'render.modes': ['human', 'rgb_array']}
 
     def __init__(self, grid_shape=(15, 15), n_agents=5, n_opponents=5, init_health=3, full_observable=False,
-                 step_cost=0, max_steps=100):
+                 step_cost=0, max_steps=100, step_cool=1, render=True):
         self._grid_shape = grid_shape
         self.n_agents = n_agents
         self._n_opponents = n_opponents
         self._max_steps = max_steps
+        self._step_cool = step_cool + 1
         self._step_cost = step_cost
+        self._render = render
```

Comment:
I don't see it being used anywhere.

Reply:
Yes, the "_render" is only for personal usage. I have removed this unused attribute.